### PR TITLE
Better TTS voice test line

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

Changes the TTS test sentence to a longer, better one

## Why It's Good For The Game

It has a wider range of sounds that give an overall better impression of the voice, probably

## Changelog
:cl:
spellcheck: The spoken line for testing TTS voices has been improved
/:cl:

### CODE REVIEW

File: code/modules/client/preferences/middleware/tts.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	var/speaker = preferences.read_preference(/datum/preference/choiced/voice)
 	var/pitch = preferences.read_preference(/datum/preference/numeric/tts_voice_pitch)
 	COOLDOWN_START(src, tts_test_cooldown, 0.5 SECONDS)
-	INVOKE_ASYNC(SStts, TYPE_PROC_REF(/datum/controller/subsystem/tts, queue_tts_message), user.client, "Hello, this is my voice.", speaker = speaker, pitch = pitch, local = TRUE)
+	INVOKE_ASYNC(SStts, TYPE_PROC_REF(/datum/controller/subsystem/tts, queue_tts_message), user.client, "I used to be a space man like you, then I took a golden bolt in the knee.", speaker = speaker, pitch = pitch, local = TRUE)
```
Comment (GoldenAlpharex) [inline code/modules/client/preferences/middleware/tts.dm:17]: Why not go for something a bit more thorough, while you're at it?
```suggestion
	INVOKE_ASYNC(SStts, TYPE_PROC_REF(/datum/controller/subsystem/tts, queue_tts_message), user.client, "The quick brown fox jumps over the lazy dog.", speaker = speaker, pitch = pitch, local = TRUE)
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: code/modules/client/preferences/middleware/tts.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply code/modules/client/preferences/middleware/tts.dm]: We went for something less obscure after all.

**Comment by Likteer** [comment]
`INVOKE_ASYNC(SStts, TYPE_PROC_REF(/datum/controller/subsystem/tts, queue_tts_message), user.client,` **"I used to be a space man like you, then I took a toolbox in the knee.",** `speaker = speaker, pitch = pitch, local = TRUE)`

Sir, it's been 12 years

**Comment by YakumoChen** [comment]
not when they release a new version every year

**Comment by Watermelon914** [comment]
this isn't necessarily better, the cost of having a longer sentence is that it takes longer to find a voice you like or want.

**Comment by YakumoChen** [comment]
Alas, humour is subjective

**Comment by Likteer** [comment]
Self-close. Commendable.

**Comment by Neskah** [comment]
Nah.

**Pull request closed without merging**